sbt

```diff
@@ -229,8 +231,13 @@ execRunner () {
     vlog ""
   }
 
-  [[ -n "$batch" ]] && exec </dev/null
-  exec "$@"
+  setTrapExit
+
+  if [[ -n "$batch" ]]; then
+    "$@" < /dev/null
+  else
+    "$@"
```

Comment:
Based on the manner in which exec was introduced (that is, without any motivation other than it seemed like it could be) I am completely comfortable removing it.